src/style.css

```diff
@@ -1 +1,81 @@
-/* Styles go here */
+
+* {
+  box-sizing: border-box;
+}
+
+.home-image-link {
+  height: 40px;
+  width: 40px;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+  color: black;
+  font-weight: 500;
+}
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 1200px;
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  list-style: none;
+  flex-wrap: wrap;
+  align-items: center;
+}
+
+.nav-item {
+  margin-right: 20px;
+  list-style: none;
+  position: relative;
+  text-transform: uppercase;
+  text-decoration: none;
+}
+
+.nav-item:last-child {
+  margin-right: 0;
+}
+
+.nav_tag {
+  display: inline-block;
+  margin: 0;
+  text-decoration: none;
+  font-size: 12px;
+  padding: 22.5px 0;
+  color: black;
+}
+
+.nav_tag:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+}
+
+.is-active:after {
+  position: absolute;
+  content: "";
+  height: 4px;
+  bottom: -1px;
+  background: #00acdc;
+  left: 0;
+  right: 0;
+  width: 100%;
```

Comment:
you have already indirectly specified width with positioning after element so need to give add width here